index.js

```diff
@@ -1 +1,2 @@
-alert('Hola usuario!');
+const nombreUsuario = prompt('Ingrese su nombre');
+alert(`Hola ${nombreUsuario}!`);
```

Comment:
A modo de ejemplo, test: En vez de utilizar interpolación de strings, se puede usar cocatenación.

```suggestion
alert('Hola !' + nombreUsuario);
```